ale_linters/json/vscodejson.vim

```diff
@@ -0,0 +1,14 @@
+" Author: Dalius Dobravolskas <dalius.dobravolskas@gmail.com>
+" Description: VSCode json languageserver
+
+function! ale_linters#json#vscodejson#GetProjectRoot(buffer) abort
```

Comment:
This is not good idea. This means that for every JSON file open in vim a new vscode-json-languageserver will be spawn.  Better look for generic project files such as the presence of a .git folder to set as project root.

Reply:
OK. Changed to git root while I think opening it for every JSON is OK as well.